src/lib/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface Room {
+  id: string;
+  name: string;
+}
```

Comment:
Tu peux rajouter aussi si tu veux, sauf si flemme
```suggestion
export interface Room {
  id: string;
  name: string;
  capacity: number;
}
```